fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogFragment.kt

```diff
@@ -225,21 +257,58 @@ class TranslationsDialogFragment : BottomSheetDialogFragment() {
                     }
                     dismiss()
                 },
-                onFromSelected = {
+                onFromSelected = { fromLanguageSelected ->
```

Comment:
```suggestion
                onFromSelected = { fromLanguage ->
```
Sometimes it's just better to just name it to what the named parameter is expecting.

Reply:
Done